v2/samples/cache/v1api/v1api20230401_redis.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: cache.azure.com/v1api20230401
```

Comment:
I'd suggest separating these samples out in their own folders like `v1api20230401` and `v1api20201201`

Reply:
Not every group does this actually, and because there's a bunch of API versions coexisting (because of redis + redisenterprise) it felt better to put it all into 1 folder like we do for compute so that people don't have to guess which API versions support which resources. Basically I think people are resource-centric and making them choose API versions first doesn't help in sample discovery. It's fine when every resource is in every API version (pick any one and you're right), but in this case not every resource is in every APIversion so it's harder.